src/crab/array_domain.cpp

```diff
@@ -1,4 +1,4 @@
-
+// This file is eBPF-specific, not derived from CRAB.
```

Comment:
This domain is implemented in Crab (there only in a header file: [domains/array_expansion.hpp](https://github.com/seahorn/crab/blob/master/include/crab/domains/array_expansion.hpp)) though it was designed specifically for eBPF-style register spilling. The implementation looks quite different due to extensive refactoring, but the logic is essentially the same. I'm not sure about licensing, but I did not implement the logic from scratch.

Reply:
Got it, removed comment